docs/proposals/20220825-azuremanaged-cluster-v1.md

```diff
@@ -0,0 +1,67 @@
+---
+title: AzureManagedCluster graduation to v1 from experimental
+authors:
+  - "@jackfrancis"
+reviewers:
+  - @CecileRobertMichon
+  - @zmalik
+  - @NovemberZulu
+creation-date: 2022-08-25
+last-updated: 2022-08-25
+status: implementable
+see-also:
+- https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2204
+- https://github.com/kubernetes-sigs/cluster-api/pull/6988
+---
+
+
+# AzureManagedCluster graduation to v1 from experimental
+
+## Summary
+
+`AzureManagedCluster` is a capz-native implementation of Azure Managed Kubernetes (AKS). Because there is no standard set of Cluster API resource definitions for a "Managed Kubernetes cluster", it is left up to the provider to re-use the existing Cluster API specification (for example, the `Cluster` and its to-be-implemented-by-provider properties such as `ControlPlaneEndpoint`, `ControlPlaneRef` and `InfrastructureRef`). As a result, capz implemented `AzureManagedCluster` with an API contract designation of "experimental", to allow for rapid prototyping and discovery.
+
+With the recent adoption of `AzureManagedCluster` by the capz community for practical, real-world use, we want to identify the set of outstanding items that may prevent graduation to v1, and address each one of them, so that future adoption can be unlocked, and users can confidently build resilient systems on top of a stable API.
+
+## Motivation
+
+### Goals
+- Agree upon a definitive v1 specification of `AzureManagedCluster`
+- Prioritize timeliness of a v1 definition so that users can confidently commit to v1 in the near term
+- Prioritize "architectural affinity" with other Cluster API providers implementing Managed Kubernetes
+
+### Non-Goals / Future Work
+- Add features to `AzureManagedCluster`
+- Standardize any opinions about how best to use AKS
+- Promote capz-specific opinions about Managed Kubernetes across the Cluster API provider ecosystem
+- Include operational support in the scope of a v1 definition
```

Comment:
Again, what does this mean? Are we redefining what "v1" means and if so is it different from semver or API version v1 ?

Reply:
removed